programs/pkey/rsa_genkey.c

```diff
@@ -70,6 +73,14 @@ int main( void )
     FILE *fpriv = NULL;
     const char *pers = "rsa_genkey";
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 122 and onwards, are displaying it without the negative. We should be self-consistent within the same program.